src/libraries/apicompat/ApiCompat.proj

```diff
@@ -35,60 +36,50 @@
   <Target Name="RunApiCompat"
           DependsOnTargets="FindReferenceAssembliesForReferences"
           AfterTargets="Build"
-          Inputs="@(ReferencePathWithRefAssemblies);$(ApiCompatNSBaselineFile);$(ApiCompatNSOnlyBaselineFile);$(PreviousNetCoreAppBaselineFile);@(ApiCompatExcludeAttributeFile)"
+          Inputs="@(ReferencePathWithRefAssemblies);$(ApiCompatNetStandard20BaselineFile);$(ApiCompatNetStandard21BaselineFile);$(ApiCompatNetCoreAppLatestStableBaselineFile);@(ApiCompatExcludeAttributeFile)"
           Outputs="$(IntermediateOutputPath)$(TargetArchitecture)-marker.txt">
-    <PropertyGroup>
-      <ApiCompatResponseFile>$(IntermediateOutputPath)apicompat.rsp</ApiCompatResponseFile>
-      <ApiCompatExitCode>0</ApiCompatExitCode>
-      <!-- TODO: Replace with ReferencePathWithRefAssemblies when ApiCompat supports passing in a list of assemblies: https://github.com/dotnet/arcade/issues/8394. -->
-      <ApiCompatArgs>--impl-dirs "$(LibrariesAllRefArtifactsPath.TrimEnd('\/'))"</ApiCompatArgs>
-      <ApiCompatArgs Condition="'@(ApiCompatExcludeAttributeFile)' != ''">$(ApiCompatArgs) --exclude-attributes "@(ApiCompatExcludeAttributeFile->Metadata('FullPath'), ',')"</ApiCompatArgs>
-    </PropertyGroup>
-
-    <WriteLinesToFile File="$(ApiCompatResponseFile)"
-                      Lines="$(ApiCompatArgs)"
-                      Overwrite="true" />
+    <Message Text="ApiCompat -> Comparing $(NetCoreAppCurrent) reference assemblies against .NETStandard2.x and .NETCoreApp$(NetCoreAppLatestStableVersion)..." Importance="high"  />
 
-    <!-- In order to update the .NETStandard baseline, you can just start the build with /p:UpdateNETStandardBaseline=true -->
-    <PropertyGroup>
-      <NetStandardLibrary20RefPath>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'netstandard.library', '$(NetStandardLibraryVersion)', 'build', 'netstandard2.0', 'ref'))</NetStandardLibrary20RefPath>
-      <NetStandard21OnlyRef>$(NETStandard21RefPath)netstandard.dll</NetStandard21OnlyRef>
-      <NetStandard21BaselineModifer>--baseline</NetStandard21BaselineModifer>
-      <NetStandard21BaselineModifer Condition="'$(UpdateNETStandardBaseline)' == 'true'">&gt;</NetStandard21BaselineModifer>
```

Comment:
The three comparisons used to use three different update flags, now they share one.

Do all three tasks run and report errors before terminating?  If not, it feels surprising that you get an error on the first one, run the update command and "boom" three files got edited.  ("But where'd the other two come from? Only one had errors...").

Not saying I disagree with collapsing them, just that it seems potentially surprising.

Reply:
The baseline files for netstandard2.0 and netstandard2.1 haven't been updated for years. Updating them all-together makes it less likely that some of them get stale. Note that there's an ApiCompat switch that removes baseline entries that don't apply anymore and we should probably leverage it in a follow-up command. Apart from that, the .NETStandard baseline files usually don't change much. I expect the "latest stable .NETCoreApp" baseline file to be way noisier.